contributing/documentation/standards.rst

```diff
@@ -137,8 +144,17 @@ Files and Directories
 English Language Standards
 --------------------------
 
-* **English Dialect**: use the United States English dialect, commonly called
-  `American English`_.
+Symfony documentation uses the United States English dialect, commonly called
+`American English`_.
+
+Unlike most popular languages, the English language doesn't have a central
+language authority and there is no official grammar or dictionary. Symfony
+documentation uses the `American English Oxford Dictionary`_ to resolve disputes
+over common words. In case a technical word isn't included in the dictionary yet,
+Symfony Documentation maintainers will decide over the disputes.
```

Comment:
-1 on this paragraph, I don't think it add much extra value. It is already clear that the docs use the American English dialect and the fact that English doesn't have a central language authority isn't really important.

However, +1 for including a link to the AE Oxford Dictionary